_data/publications.yml

```diff
@@ -175,6 +175,7 @@
   slug: gibson-mortality
   authors: Gibson GC, Reich NG, Sheldon D
   preprint: https://www.medrxiv.org/content/10.1101/2020.12.22.20248736v2
+  pdf: https://www.medrxiv.org/content/10.1101/2020.12.22.20248736v2.full.pdf
```

Comment:
Could we make this PDF link to the published article PDF, not the preprint PDF?

Reply:
Hi Nick. I sent a separate message about this, since I cannot access the published article PDF.

Reply:
Thanks for the link - the article was updated.